_includes/18f-example-header.html

```diff
@@ -0,0 +1 @@
+<h2>Example from 18F</h2>
```

Comment:
Utilizing includes for the headers so we can easier update the headers in the future